### PR TITLE
Add Charge conjugate methods to decay-related classes

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def get_version():
         'pathlib2>=2.3; python_version<"3.5"',
         'enum34>=1.1; python_version<"3.4"',
         'importlib_resources>=1.0; python_version<"3.7"',
+        'cachetools',
```
Comment (henryiii) [inline setup.py]: Let me see if I can do this by using the stdlib version if the user is on Python 3.

Comment (eduardo-rodrigues) [reply setup.py]: Else we can also remove the caching for now, as it is not a game-changer, at least for now. I have no strong feeling here.

This being said, I may use cachetools in other places soon ...

Comment (henryiii) [reply setup.py]: If you use it in other places, we can move this to a utility file, and then make it a lambda so you can call `cacher(64)` and that will return the correct object to make into a decorator. (Assuming this works)

Comment (eduardo-rodrigues) [reply setup.py]: Yes, something to have in mind.

**Pull request merged**